src/Components/ProgressiveOnboarding/ProgressiveOnboardingAlertCreateSimple.tsx

```diff
@@ -7,9 +7,9 @@ import {
 } from "Components/ProgressiveOnboarding/withProgressiveOnboardingCounts"
 import { useSystemContext } from "System/SystemContext"
 import { useDismissibleContext } from "@artsy/dismissible"
-import { PROGRESSIVE_ONBOARDING_ALERTS } from "Components/ProgressiveOnboarding/progressiveOnboardingAlerts"
+import { PROGRESSIVE_ONBOARDING } from "Components/ProgressiveOnboarding/progressiveOnboardingKeys"
 
-const ALERT_ID = PROGRESSIVE_ONBOARDING_ALERTS.alertCreate
```

Comment:
When this [got extracted into it's own package](https://github.com/artsy/force/pull/13310) — things got renamed and unfortunately they were named as 'alerts' which conflicts with the notion of these being used for capital-A alerts. So I renamed everything to make logical sense again.